app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyTextFieldQuestionScreen.kt

```diff
@@ -0,0 +1,106 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
+import com.kks.nimblesurveyjetpackcompose.ui.presentation.common.SurveyText
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White30
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White40
+import kotlinx.coroutines.delay
+
+private const val TEXT_FIELD_HEIGHT = 56
+private const val SPACE_BETWEEN_TEXT_FIELDS = 16
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun SurveyTextFieldQuestionScreen(
+    answers: List<SurveyAnswer>,
+    onAnswerChange: (answers: List<SurveyAnswer>) -> Unit
+) {
+    var answerStates by remember { mutableStateOf(answers.map { it.answer }) }
+    val heightOfQuestions = (TEXT_FIELD_HEIGHT * answers.size) + (SPACE_BETWEEN_TEXT_FIELDS * (answers.size - 1))
```

Comment:
I'm not sure I understand why we need this?
We could just set `singleLine = true` on the `TextField` below and remove the modifier from the `Column`, surely that would have the same effect without having to do any calculations or declare anything extra? 🤔

Reply:
@ryan-conway I want to have even space between each text field and also have the same size of space as described in the Figma. That's why I calculate the height of the column for the number of questions and the required number of spaces here.

Reply:
@kaungkhantsoe surely we can use `Modifier.padding(below = 16.dp)`? 
Or if we don't want final item padding `Modifier.padding(below = if (index == answers.lastIndex) 0.dp else 16.dp)`?

Reply:
@ryan-conway I tried to run `Modifier.padding(below = if (index == answers.lastIndex) 0.dp else 16.dp)` and it show as below, it doesn't work.
![Screenshot_20221024_163731](https://user-images.githubusercontent.com/32578035/197497212-b4508e4d-3851-4005-b146-9dac02c8f402.png)

Reply:
Also the calculation here is executed only once for each recomposition and it is also simple calculation which doesn't take much time. I think it is acceptable. What do you think?

Reply:
I will use `spaceBy()`